treeherder/model/sql/template_schema/project.sql.tmpl

```diff
@@ -153,6 +153,33 @@ CREATE TABLE `job` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
```

Comment:
you should either put this table in the reference data schema or remove the repository column as it is redundant